src/main/java/io/github/sefiraat/crystamaehistoria/slimefun/items/tools/SpiritualSilken.java

```diff
@@ -70,7 +71,7 @@ public ItemUseHandler getItemHandler() {
         return e -> {
             e.cancel();
             Optional<Block> blockOptional = e.getClickedBlock();
-            if (blockOptional.isPresent()) {
+            if (blockOptional.isPresent() && Slimefun.getProtectionManager().hasPermission(e.getPlayer(), blockOptional.get(), Interaction.BREAK_BLOCK)) {
```

Comment:
Check is, ofc, a good idea but can it please be added after the existing `blockOptional.get()` as a guard statement?  Possibly with it’s own message talking about no permissions?

Reply:
Sure one second 👍